CHANGELOG.md

```diff
@@ -0,0 +1,314 @@
+# Fastlane CHANGELOG
+
```

Comment:
@AliSoftware yes I meant `Fastfile`. You can also change this: https://github.com/fastlane/fastlane/blob/master/fastlane/Fastfile#L112